docker-for-mac/index.md

```diff
@@ -159,28 +159,50 @@ to decrease it, lower the number.
 
 **Swap**: Configure swappiness as needed. The default is 1.
 
-### Disk tab
+### Disk
 
 Specify the **Disk image location** of the Linux volume, where containers and images are stored.
 
 You can also move the disk image location. If you attempt to move the disk image
 to a location that already has one, you get a prompt asking if you want to use
 the existing image or replace it.
 
-![Proxies settings](/docker-for-mac/images/menu/d4m-menu-prefs-disk.png){:width="400px"}
+![Disk settings](/docker-for-mac/images/menu/d4m-menu-prefs-disk.png){:width="400px"}
 
-### Proxies tab
+### Proxies
 
-Docker for Mac detects HTTP/HTTPS Proxy Settings and automatically propagate
-these to Docker and to your containers. For example, if you set your proxy
-settings to `http://proxy.example.com`, Docker uses this proxy when pulling
-containers.
+Docker for Mac detects HTTP/HTTPS Proxy Settings from macOS and automatically
+propagates these to Docker and to your containers. For example, if you set your
+proxy settings to `http://proxy.example.com`, Docker uses this proxy when
+pulling containers.
 
 ![Proxies settings](/docker-for-mac/images/menu/d4m-menu-prefs-proxies.png){:width="400px"}
 
+![macOS Proxy Settings](images/proxy-settings.png){:width="400px"}
+
+When you start a container, your proxy settings propagate into
+the containers. For example:
+
+```
+$ docker run -it alpine env
+PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+HOSTNAME=b7edf988b2b5
+TERM=xterm
+HOME=/root
+HTTP_PROXY=http://proxy.example.com:3128
+http_proxy=http://proxy.example.com:3128
+no_proxy=*.local, 169.254/16
+```
+
+You can see from the above output that the `HTTP_PROXY`, `http_proxy`, and
+`no_proxy` environment variables are set. When your proxy configuration changes,
+Docker restarts automatically to pick up the new settings. If you have
```

Comment:
I thought VPNKit was able to catch thse changes without restarting, can you double-check it restarts when changing the system proxies @akimd ?

Reply:
I have tried to change my system Proxy, and not restart Docker for Mac.  Then I have:
```
$ docker run -it alpine env
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
HOSTNAME=83bdc8002344
TERM=xterm
HOME=/root
```
Then I have restarted Docker for Mac, and... same result.

So I don't know what is going wrong here (maybe just the doc), but this is probably that should be dealt with in another issue.  Let's apply this one anyway (the piece of doc is not new, it's just moved).